client.c

```diff
@@ -956,6 +956,7 @@ static void convert_ts(struct thread_stat *dst, struct thread_stat *src)
 	dst->error		= le32_to_cpu(src->error);
 	dst->thread_number	= le32_to_cpu(src->thread_number);
 	dst->groupid		= le32_to_cpu(src->groupid);
+	dst->job_start = le64_to_cpu(src->job_start);
```

Comment:
Can you line this up like the other lines?

Reply:
Sure, sorry about that.